app/controllers/items_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ItemsController < ApplicationController
+
+  def index
+    @items = Item.all
+  end
+
+  def show
+    @item = Item.find(params[:id])
+  end
+
+  def new
+    @item = Item.new
+  end
+
+  def create
+    @item = Item.new(item_params)
+    if @item.save
+      flash[:success] = "You have added a new resource item to the library!"
+      redirect_to '/items/new'
```

Comment:
Typically after creating something, we redirect to the "show" page for that object. If we'd rather go back to the "new" form instead that's fine too, just wanted to mention it as something that might feel more intuitive for the user.